sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -22,64 +22,64 @@ Sets or updates one or more properties' values for a site collection.
 
 ```powershell
 Set-SPOSite [-Identity] <SpoSitePipeBind> [-AllowSelfServiceUpgrade <Boolean>] [-Confirm]
```

Comment:
Thanks for sorting all these existing params.
Could you put the "-Allow...." params together?